tests/core/asyncio/test_asyncio_subscriptions_api.py

```diff
@@ -194,7 +194,7 @@ def noop(event):
     asyncio.ensure_future(server_a.subscribe(WaitSubscription, noop))
 
     await asyncio.wait_for(
-        client.wait_until_any_remote_subscribed_to(WaitSubscription), timeout=0.1
+        client.wait_until_any_remote_subscribed_to(WaitSubscription), timeout=1
```

Comment:
Out of curiosity: Any particular reason why this increase is needed?

Reply:
I think it just snuck in from some debugging changes.  Should be reverted.